libnymea-zigbee/zigbee.h

```diff
@@ -55,6 +55,7 @@ class Zigbee
         ZigbeeProfilePersonalHomeHospitalCare = 0x0108,
         ZigbeeProfileAdvancedMetering = 0x0109,
         ZigbeeProfileLightLink = 0xC05E,
+        ZigbeeProfileDevelco = 0xC0C9,
```

Comment:
Weird... Seems strange to have manufacturer specific stuff at this place...

Reply:
This way the entire stack knows what profile this is, not just the final user already knowing it is a develco node (providing as much information as possible). It's not an official profile, but a well known one.